Dockerfile

```diff
@@ -453,6 +454,20 @@ ENV PATH "$PATH:/opt/buildhome/.dotnet"
 ENV DOTNET_ROOT "/opt/buildhome/.dotnet"
 #populate local package cache
 RUN dotnet new
+
+
+################################################################################
+#
+# Swift
+#
+################################################################################
+USER buildbot
+ENV SWIFTENV_ROOT="/opt/buildhome/.swiftenv"
+RUN git clone --depth 1 https://github.com/kylef/swiftenv.git "$SWIFTENV_ROOT"
+ENV PATH "$SWIFTENV_ROOT/bin:$SWIFTENV_ROOT/shims:$PATH"
+RUN swiftenv install 5.1.3
```

Comment:
@mraerino I'd like to point out here is where we preinstall the current stable version of Swift (via Swiftenv), which is v5.1.3.